frontend/src/components/OdhCard.scss

```diff
@@ -100,7 +100,8 @@
       justify-content: flex-end;
     }
 
-    &__link {
+    & &__link {
+      font-size: var(--pf-v5-global--FontSize--md);
```

Comment:
When changing the cards to use PF `Button` the font sizes were reduced down to the default `sm` font size due to the override in `App.scss`. This change increases the font size for the footer links and adjusts the specificity so that it can take priority over the that from `App.scss`.